packages/jest-validate/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "jest-validate",
+  "version": "18.1.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/facebook/jest.git"
+  },
+  "license": "BSD-3-Clause",
+  "main": "build/index.js",
+  "dependencies": {
+    "chalk": "^1.1.1",
+    "pretty-format": "^18.1.0"
+  },
+  "scripts": {
+    "test": "../../packages/jest-cli/bin/jest.js"
+  }
```

Comment:
actually we don't need this scripts section any more in any of the packages. Just noticed :O